code/web/src/modules/user/api/actions.js

```diff
@@ -116,3 +114,38 @@ export function getGenders() {
     }))
   }
 }
+
+// Submit New Address
+export function submitAddress(id, address) {
+  // return dispatch => {
+    return axios.post(routeApi, mutation({
+      operation: 'updateAddress',
+      variables: {id, address},
+      fields: ['id', 'address']
+    }))
+    .then(response => {
+      console.log(response, 'RESPONSE IN ACTIONS')
+      const user = response.data.data.userAddress
+      const token = window.localStorage.getItem('token')
+      dispatch(setUser(token, user))
+    })
+  // }
+}
+
+// Update User
+
+export function updateUser(user) {
+  return dispatch => {
+    return axios.post(routeApi, mutation({
+      operation: 'updateUser',
+      variables: user,
+      fields: ['id', 'email', 'address', 'image', 'description']
+    }))
+    .then(response => {
+      console.log(response, 'what the heck')
+      const user = response.data.data.updateUser
+      const token = window.localStorage.getItem('token')
+      dispatch(setUser(token, user))
+    })
+  }
+}
```

Comment:
Nice job on this one, this actually makes the post to the mutation and gets returned the user information from the database